### DISCUSSION TITLE
v0.107.17 update on Windows won't start

### DISCUSSION THREAD

**Discussion by RealAmes** [AdguardTeam/AdGuardHome#5097] [Q&A]
I tried to update to v0.107.17 on several Windows computers and the service won't start anymore.

**Comment by unknown4849** [comment]
Same , first i think just my self...

try make new configuration file and still same Services stop immediately , and use powershell (start manually) still same and got this message : 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x8 pc=0x9e6738]

btw i try to v0.107.16 , but strange like connection lost ( DNS_PROBE_POSSIBLE ) like Services not respond for a while , yeah i try update via this version and same problem services stop immediately

**Comment by andekande** [comment]
v0.107.18 fixed this - you need to manually download it.
